src/gui/editorwidgets/qgsrelationreferencewidget.h

```diff
@@ -269,6 +268,13 @@ class GUI_EXPORT QgsRelationReferenceWidget : public QWidget
     //! unset the currently related feature
     void deleteForeignKeys();
 
+    /**
+      * Trigger save of the embedded referenced attribute form.
+      *
+      * \since QGIS 3.42
+      */
+    bool saveReferencedAttributeForm();
```

Comment:
can you specify the return value, and state that it will return TRUE if no embed form is present